source/includes/minimum-destination-privileges.rst

```diff
@@ -0,0 +1,112 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 20
+
+   * - Sync Type
+     - Minimum Destination Privileges
+
+   * - Default
+     - .. code-block:: javascript
+       
+          [
+	        {
+		        "resource": { "cluster": true },
```

Comment:
[spacing]

These code block sections have some spacing issues throughout. We should use three spaces to indent, and the commas should follow the fields immediately. I provided an example below:

```
{
   "resource": { "cluster": true },
      "actions": [
         "appendOplogNote",
         "getDefaultRWConcern",
         "getShardMap",
         "hostInfo",
         "listDatabases",
         "listShards", 
         "replSetGetConfig",
	 "replSetGetStatus",
         "serverStatus"
       ]
},